UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Objects.Security;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Systems
+{
+	/// <summary>
+	/// Contains a list of all mainstation crewmembers, including traitors and silicons but not, for example, fugitives or wizards.
+	/// </summary>
+	public class CrewManifestManager : MonoBehaviour
+	{
+		public static CrewManifestManager Instance;
+
+		/// <summary>
+		/// A list of all mainstation crewmembers, including traitors and silicons but not, for example, fugitives or wizards.
+		/// </summary>
+		public List<CrewManifestEntry> CrewManifest { get; private set; } = new List<CrewManifestEntry>();
+		/// <summary>
+		/// A list of all security records. By default, includes traitors and silicons but not, for example, fugitives or wizards.
+		/// Records can be updated, added or removed during gameplay at the security records console.
+		/// </summary>
+		public List<SecurityRecord> SecurityRecords { get; private set; } = new List<SecurityRecord>();
+
+		#region Lifecycle
+
+		private void Awake()
+		{
+			if (Instance == null)
+			{
+				Instance = this;
```

Comment:
Codacy found an issue: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6384425)